tools/build_framework.sh

```diff
@@ -10,7 +10,8 @@ fi
 
 FRAMEWORK_NAME=$1
 FRAMEWORK_DIR=$2
```

Comment:
how about having the args to build_framework.sh:
```
FRAMEWORK_DIR=$2
EXECUTOR_DIR=$3 # path to executor to build (parallels FRAMEWORK_DIR), or an empty string ("" will register as an arg for this indexing)
PUBLISH_STEP=$4
```

if EXECUTOR_DIR is empty then no additional work, or if populated then run `gradlew -p ${EXECUTOR_DIR}` and also include `${EXECUTOR_DIR}/build/distributions/*` in the publish step at the end